staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/promise/interface.go

```diff
@@ -16,27 +16,85 @@ limitations under the License.
 
 package promise
 
-// Mutable is a variable that is initially not set and can be set one
-// or more times (unlike a traditional "promise", which can be written
-// only once).
-type Mutable interface {
+// This file defines interfaces for promsies and futures and related
+// things.
 
+// Readable is a variable that is initially not set and later becomes
```

Comment:
variable: it is an interface.

Please rephrase

Reply:
Done.